UM/Math/Polygon.py

```diff
@@ -22,16 +23,13 @@ def approximatedCircle(radius):
         :return: A polygon that approximates a circle.
         """
 
-        return Polygon(points = numpy.array([
-            [-radius, 0],
-            [-radius * 0.707, radius * 0.707],
-            [0, radius],
-            [radius * 0.707, radius * 0.707],
-            [radius, 0],
-            [radius * 0.707, -radius * 0.707],
-            [0, -radius],
-            [-radius * 0.707, -radius * 0.707]
-        ], numpy.float32))
+        step = 2 * math.pi / num_segments
+
+        points = []
+        for i in range(0, num_segments):
+            points.append([radius * -math.cos(i * step), radius * math.sin(i * step)])
```

Comment:
Not really performance-sensitive here, but for next time, Numpy supports vector operations like so:

```
points = numpy.arange(0, 2 * math.pi, step)
points = numpy.stack((numpy.cos(points), numpy.sin(points)))
```